packages/viewer/src/interactions/interactionApi.ts

```diff
@@ -191,6 +195,33 @@ export class InteractionApi {
     return this.currentCamera != null;
   }
 
+  /**
+   * Scales the provided `pixelThreshold` by the device's pixel ratio
+   * and the configured `coarsePointerThresholdScale` value if the
+   * primary pointer input is coarse (touch).
+   *
+   * Scales the provided `pixelThreshold` by the configured
+   * `finePointerThresholdScale` value if the primary pointer input is
+   * fine (mouse).
+   *
+   * @param pixelThreshold - The pixel threshold to scale.
+   * @returns The scaled pixel threshold.
+   */
+  public scalePixelThreshold(
+    pixelThreshold: number,
+    isTouch?: boolean
+  ): number {
+    if (this.isCoarseInputDevice(isTouch)) {
+      return (
+        pixelThreshold *
+        window.devicePixelRatio *
+        this.getConfig().coarsePointerThresholdScale
+      );
+    } else {
+      return pixelThreshold * this.getConfig().finePointerThresholdScale;
```

Comment:
I think we want to take into account the device pixel ratio even for fine pointers. My guess is that a stylus on a iPad or windows desktop would be categorized as a fine pointer device.

Reply:
I believe a stylus actually registers as a `coarse` input type, and from what I've read only a mouse or trackpad would register as a `fine` input type. Certainly not opposed to changing this, but it would mean that on macbooks with retina display we'd be scaling up the threshold (which maybe we want anyway)